flow-typed/publish.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+declare type UpdatePublishFormData = {
```

Comment:
Naming is a little inconsistent between these two types.

Also, you may be able to use Intersection Types to avoid repeating the shared values across both (but I've never used this feature).

Reply:
Yeah... the publish types were one of the first added. Now that I have a better idea of flow and have developed some patterns in other parts of the app this can be cleaned up a lot. Planning to clean it up when I add the tags to the publish page